tasks/copy_to_glacier/copy_to_glacier.py

```diff
@@ -169,20 +169,22 @@ def task(event: Dict[str, Union[List[str], Dict]], context: object) -> Dict[str,
     sqs_body["provider"] = {}
     sqs_body["collection"] = {}
     sqs_body["granule"] = {}
+    sqs_body["provider"][
+        "providerName"
+    ] = None  # set to None because cumulus only returns providerId for now. In case it is available in the future, update orca_copy_to_glacier_workflow.asl.json and config.json as needed
+    sqs_body["provider"]["providerId"] = config["providerId"]
+    sqs_body["collection"]["shortname"] = config["collection_shortname"]
+    sqs_body["collection"]["version"] = config["collection_version"]
+    sqs_body["collection"]["collectionId"] = (
+        config["collection_shortname"] + "___" + config["collection_version"]
+    )  # Cumulus currently creates collectionId by concating shortname + ___ + version See https://github.com/nasa/cumulus-dashboard/blob/18a278ee5a1ac5181ec035b3df0665ef5acadcb0/app/src/js/utils/format.js#L342
```

Comment:
Again, this is very long. May want to think of putting the comment above the assignment as a multi line comment for better readability.

Reply:
updated